app/controllers/v0/medical_copays_controller.rb

```diff
@@ -8,6 +8,10 @@ def index
       render json: vbs_service.get_copays
     end
 
+    def get_pdf_statements_by_id
```

Comment:
Do you have swagger doc additions for the new endpoint?

Reply:
@LindseySaari Good call out will add them 🙂

Reply:
Swagger docs and accompanying swagger test have been added